### DISCUSSION TITLE
Savefiles on Linux

### DISCUSSION THREAD

**Discussion by Nekotje** [mtkennerly/ludusavi#254] [General]
i use this to make copies of my games and im very happy with it. It works pretty good.

Only 1 thing i noticed with 2 games might be more.
![image](https://github.com/mtkennerly/ludusavi/assets/6666518/a1db30f4-f444-41f3-8035-f3c63a3c2e14)
These are my roots where my games are installed.

with the first root entry
/home/User/.steam/debian-installation/
i have a game installed ship graveyard simulator and ship graveyard simulator 2
When i click preview that it does a scan these dont show up in the list.

But when i look at where the save game is
/home/User/.steam/debian-installation/steamapps/compatdata/2201940/pfx/drive_c/users/steamuser/AppData/Local/ShipGraveyard2/Saved/SaveGames/FullSave_AutoSave_001.sav

Its in that Root directory but it does not find it.

**Comment by mtkennerly** [comment]
Hi! I noticed you also opened https://github.com/mtkennerly/ludusavi/issues/255, so let's focus the conversation there :)
